README.md

```diff
@@ -1,8 +1,8 @@
-# tap-facebook
+# `tap-facebook`
 
-`tap-facebook` is a Singer tap for facebook.
+facebook tap class.
```

Comment:
```suggestion
Singer tap for extracting data from the Facebook Marketing API.
```

Reply:
Good catch! I refreshed it in a separate commit because there were a few descriptions that werent reflected either.